docs/integrations/sources/e2e-test-cloud.md

```diff
@@ -0,0 +1,34 @@
+
+The single-stream catalog config exists just for convenient, since in many testing cases, one stream is enough. If only one stream is specified in the multi-stream catalog config, it is equivalent to a single-stream catalog config.
+
+Here is its configuration:
+
+| Mock Catalog Type | Parameters          | Type    | Required | Default             | Notes                                                                                                                                                   |
+| ----------------- | ------------------- | ------- | -------- | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Single-stream     | stream name         | string  | yes      |                     | Name of the stream in the catalog.                                                                                                                      |
+|                   | stream schema       | json    | yes      |                     | Json schema of the stream in the catalog. It must be a valid Json schema.                                                                               |
+|                   | stream duplication  | integer | no       | 1                   | Duplicate the stream N times to quickly create a multi-stream catalog.                                                                                  |
+| Multi-stream      | streams and schemas | json    | yes      |                     | A Json object specifying multiple data streams and their schemas. Each key in this object is one stream name. Each value is the schema for that stream. |
+| Both              | max records         | integer | yes      | 100                 | The number of record messages to emit from this connector. Min 1. Max 100 billion.                                                                      |
+|                   | random seed         | integer | no       | current time millis | The seed is used in random Json object generation. Min 0. Max 1 million.                                                                                |
+|                   | message interval    | integer | no       | 0                   | The time interval between messages in millisecond. Min 0 ms. Max 60000 ms (1 minute).                                                                   |
+
+## Changelog
+
+The OSS and Cloud variants have the same version number. The Cloud variant was initially released at version `1.0.0`.
+
+| Version | Date       | Pull request                                             | Notes                                               |
+|---------|------------|----------------------------------------------------------|-----------------------------------------------------|
+| 2.1.2   | 2023-03-01 | [23656](https://github.com/airbytehq/airbyte/pull/23656) | Fix inheritance between e2e-test and e2e-test-cloud |
```

Comment:
[nit] I view this sort of like a strict-encrypt connector. Ideally the versions would match.
```suggestion
| 2.1.4   | 2023-03-01 | [23656](https://github.com/airbytehq/airbyte/pull/23656) | Fix inheritance between e2e-test and e2e-test-cloud |
```

Reply:
2.1.2 is right. this is for e2e-test-cloud not e2e-test. the latter is the one a 2.1.4.

Reply:
or is your goal to just have e2e-tst-cloud and e2e-test always have the same versions?

Reply:
works for me.